pkg/cloud/azure/actuators/machine/reconciler.go

```diff
@@ -210,6 +213,14 @@ func (r *Reconciler) Delete(ctx context.Context) error {
 		return errors.Wrapf(err, "Unable to delete network interface")
 	}
 
+	OSDiskSpec := &disks.Spec{
+		Name: fmt.Sprintf("%s_OSDisk", r.scope.Machine.Name),
```

Comment:
Can you please move this to pkg/azure/defaults.go, and also move the other hardcoded place virtualmachines.go as well